packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -1026,16 +1026,14 @@ function updateMemo<T>(
   return nextValue;
 }
 
-// in a test-like environment, we want to warn if dispatchAction()
-// is called outside of a batchedUpdates/TestUtils.act(...) call.
-let shouldWarnForUnbatchedSetState = false;
+let shouldWarnForUnactedSetState = false;
 
 if (__DEV__) {
   // jest isn't a 'global', it's just exposed to tests via a wrapped function
   // further, this isn't a test file, so flow doesn't recognize the symbol. So...
   // $FlowExpectedError - because requirements don't give a damn about your type sigs.
   if ('undefined' !== typeof jest) {
-    shouldWarnForUnbatchedSetState = true;
+    shouldWarnForUnactedSetState = true;
```

Comment:
I know this isn't new but this is the first time I've noticed it. How does this work for non-Jest test runners?

Reply:
Everything 'works', except we don't warn when you don't use act around an update. We considered other options (NODE_ENV=test etc), but this seemed most reasonable for now.